bHY6yh-de7365/cod-bxxqqq_bHY6yh-de7365.xml

```diff
@@ -0,0 +1,150 @@
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 26</title>
+               <date when="2023-07-01">July 1, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+              <witness xml:id="B" n="cod-bxxqqq">Paris, Bibliothèque Nationale de France, Nouv. acq. lat. 2469</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-07-01" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#B" n="13-r"/>
+            <cb ed="#B" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de7365"><!-- p1l1-r26 -->
+            <head xml:id="bHY6yh-de7365-Hd1e108">Rubrica 26</head>
+            <head xml:id="bHY6yh-de7365-Hd1e111" type="question-title">De peccatorum conuersione ad religionem.</head>
+            <p xml:id="bHY6yh-de7365-d1e114">
+                de pec<lb ed="#B" break="no"/>catorum 
+                conuersione ad religionem Rubrica 26
+            </p>
+            <p xml:id="bHY6yh-de7365-d1e125">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de7365-Rd1e122">Iob hystoria capitulo 22</ref> ¶ <quote xml:id="bHY6yh-de7365-Qd1e129">Si reuersus 
+              <lb ed="#B"/>fueris ad omnipotentem edifi<lb ed="#B" break="no"/>caberis 
+              et longe facies iniquitatem
+              <lb ed="#B"/><sic>athabernaculo</sic> tuo</quote>
+            </p>
+            <p xml:id="bHY6yh-de7365-d1e139">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de7365-Rd1e138">Thobie hystoria capitulo<!--line jumps up--> 13 et fideli<quote xml:id="bHY6yh-de7365-Qd1e143">Conuerti<!--line jumps down-->mini peccatores et 
+              <lb ed="#B"/>facite iusticiam coram deo creden<lb ed="#B" break="no"/>tes 
```

Comment:
Looks like you've lost your closing `</ref>` tag here which is making the XML invalid.
```suggestion
            <p xml:id="bHY6yh-de7365-d1e139">
              <lb ed="#B"/><ref xml:id="bHY6yh-de7365-Rd1e138">Thobie hystoria capitulo<!--line jumps up--> 13 et fideli</ref> <quote xml:id="bHY6yh-de7365-Qd1e143">Conuerti<!--line jumps down-->mini peccatores et 
              <lb ed="#B"/>facite iusticiam coram deo creden<lb ed="#B" break="no"/>tes 
```